invenio_rdm_records/ext.py

```diff
@@ -24,6 +25,15 @@ def __init__(self, app=None):
     def init_app(self, app):
         """Flask application initialization."""
         self.init_config(app)
+        self.metadata_extensions = MetadataExtensions(
```

Comment:
Shelved: Should it be plural? There are multiple fields, but it's it only one extension 😇 (trivial question)

Reply:
Good question :thinking: ... I will ask around. (will probably merge anyway...)